bin/lib/installable/edg.py

```diff
@@ -14,6 +14,14 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+class EdgBackendCompilerScrape:
```

Comment:
There are some more pythonic ways to make "structs" like this:

```py
@dataclass(frozen = True)
class EdgBackendCompilerScape:
    c_includes: str
    cpp_includes: str
    version: str
```
should be identical to the code below

(with a `from dataclasses import dataclass` at the top